### DISCUSSION TITLE
Does Rspamd support external Redis?

### DISCUSSION THREAD

**Discussion by georglauterbach** [docker-mailserver/docker-mailserver#3942] [General]
Does DMS support enabling an external Redis instance with Rspamd?

**Comment by glime-ai** [comment]
**Please note** that this response was created using generative AI, leveraging previously available information from the repository. This information may be helpful. However, it is our strong recommendation that you carefully consider the information before blindly applying it.

---

Yes, Docker Mail Server (DMS) supports enabling an external Redis instance for use with Rspamd. You can opt-out of the internal Redis service and configure Rspamd to use an external Redis instance by setting the environment variable `ENABLE_RSPAMD_REDIS=0`. Additionally, you will need to provide configuration at `/etc/rspamd/local.d/redis.conf` similar to the following:

```plaintext
servers = "redis.example.test:6379";
expand_keys = true;
```

This configuration allows you to specify the address of your external Redis server and other necessary settings.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=0d5b86fb7eea4a54acac405212c29aba&utm_medium=github).

**Comment by georglauterbach** [comment]
Test passed - nice!
